cmd/observe/observe.go

```diff
@@ -234,6 +235,12 @@ programs attached to endpoints and devices. This includes:
 		false,
 		"Display all information in numeric form",
 	)
+	observerCmd.Flags().StringVar(
+		&token,
+		"token",
```

Comment:
`token` seems rather generic for a CLI usage, we may want to add another word. `<some-qualifier>-token`